populate_casgenes.py

```diff
@@ -1,5 +1,7 @@
 import os
+
 import numpy as np
+
 # !/usr/bin/env python3
```

Comment:
This is broken (though not by you.)

Reply:
Which part? Do you want me to remove the shebang?

Reply:
The shebang must be on the first line. The encoding also should also be on the first two or three lines or something like that.

This was always broken though.

Reply:
Is this file even marked executable? Maybe no-one attempts to run it directly anyway; confirm with @mbonsma.

Reply:
Um yeah it should be executable? It's one of the one-time populate the database scripts.